bin/www

```diff
@@ -40,12 +40,19 @@ if (key_file_location !== undefined && cert_file_location !== undefined) {
        key: fs.readFileSync(key_file_location),
        cert: fs.readFileSync(cert_file_location)
     };
-    server = https.createServer(options, app);
+    server = https.createServer(options, app).listen(https_port);
+
+    http.createServer(function (req, res) {
+        var url_location = "https://" + req.headers['host'].split(":")[0] + ":" + nconf.get('HTTPS_HOST_PORT') + req.url;
```

Comment:
Use `config.get('HTTPS_HOST_PORT')` instead (new way) and move this up with the other 2 port declarations.

Also, should this only be an env var, or should it also reside in config.json? If the latter, then we'd have 3 different ports specified in our config. But since this is only used for Docker purposes, maybe we can just use it as an env var for now.

Also, it should default to the value of HTTPS_PORT if not set.